### PR TITLE
Added new language: NL (dutch)

### PR DESCRIPTION
**Pull request by maatj05**

### CODE REVIEW

File: BytexDigital.Blazor.Components.CookieConsent/CookieConsentOptions.cs [hunk 1]
```diff
@@ -162,26 +173,31 @@ public class CookieConsentOptions
...
-                    ["de"] = "Umbedingt notwendige Cookies"
+                    ["de"] = "Umbedingt notwendige Cookies",
+                    ["nl"] = "Strict noodzakelijke cookies"
                 },
                 DescriptionText = new()
                 {
                     ["en"] = "These cookies are essential for the proper functioning of this website. They do not contain personal data and are not used to track you.",
-                    ["de"] = "Diese Cookies sind umbedingt notwendig für die Nutzung dieser Webseite. Sie enthalten keine personenbezogenen Daten und werden nicht für Tracking verwendet."
+                    ["de"] = "Diese Cookies sind umbedingt notwendig für die Nutzung dieser Webseite. Sie enthalten keine personenbezogenen Daten und werden nicht für Tracking verwendet.",
+                    ["nl"] = "deze cookies zijn essentieel voor het goed functioneren van deze website. Ze bevatten geen persoonlijke gegevens en worden niet gebruikt om u te volgen."
```
Comment (RyanTT) [inline BytexDigital.Blazor.Components.CookieConsent/CookieConsentOptions.cs:183]: Is this supposed to not be capitalized?

**Review by RyanTT — COMMENTED** [review]
Thanks for the PR!

I've comment on a line where I think theres a little typo, let me know if that's the case :)

File: BytexDigital.Blazor.Components.CookieConsent/CookieConsentOptions.cs (hunk 1 above)
Comment (maatj05) [reply BytexDigital.Blazor.Components.CookieConsent/CookieConsentOptions.cs:183]: You're wright, it is supposed to be capitalized

**Comment by maatj05** [comment]
That’s nice… it’s my first PR 😊

**Pull request merged**